src/execution/sql/storage_interface.cpp

```diff
@@ -47,6 +48,23 @@ storage::ProjectedRow *StorageInterface::GetIndexPR(catalog::index_oid_t index_o
   return index_pr_;
 }
 
+bool StorageInterface::VerifyTableInsertConstraint() {
+  auto *pr = table_redo_->Delta();
+  return db_accessor_->VerifyTableInsertConstraint(table_oid_, pr);
+}
+
+bool StorageInterface::UpdateVerify() {
+    return true;
+}
+
+bool StorageInterface::UpdateCascade(storage::TupleSlot table_tuple_slot) {
+  return db_accessor_->UpdateCascade(table_oid_, table_tuple_slot);
```

Comment:
You need to add `#include "catalog/catalog_accessor.h"` at the beginning of `storage_interface.cpp`. Otherwise `UpdateCascade` cannot be found and compilation will fail.